File: app/src/main/java/org/metabrainz/mobile/api/data/search/response/RecordingSearchResponse.java
package org.metabrainz.mobile.api.data.search.response;

import org.metabrainz.mobile.api.data.search.entity.Recording;

import java.util.List;

public class RecordingSearchResponse extends JSONResponse {

    private List<Recording> recordings;

    public List<Recording> getRecordings() {
        return recordings;
    }

    public void setRecordings(List<Recording> recordings) {
        this.recordings = recordings;
    }
}

File: app/src/main/java/org/metabrainz/android/model/response/RecordingSearchResponse.kt
package org.metabrainz.android.model.response

import org.metabrainz.android.model.mbentity.Recording

class RecordingSearchResponse : SearchResponse<org.metabrainz.android.model.mbentity.MBEntity?>() {
    var recordings: List<Recording>? = null
}